assets/sass/components/feature-tours/_googlesitekit-dashboard-sharing.scss

```diff
@@ -0,0 +1,43 @@
+/**
+ * Dashboard Sharing Feature Tour styles.
```

Comment:
This is a workaround I came up with to solve an unforseen problem with Joyride which doesn't seem to support dynamically revealing a next step's `target` element while transitioning from the first. Even with the events it provides, I couldn't find one that allowed for this to happen without making the entire tour `controlled`. More details below, but this essentially tricks Joyride into thinking the target is visible while keeping it hidden, but only while the main DS tour is active.

There may be more needed here for accessibility.

Reply:
Update: seems good; doesn't pose any issues with JAWS or macOS VoiceOver.

Reply:
(Also, for posterity, `aria-hidden` if it _did_ pose an issue would work, not sure why I didn't immediately mention/think of that at the time. But it's fine without 🎉 )